pkg/atls/atls.go

```diff
@@ -247,13 +247,13 @@ func GetCertificate(caUrl string, cvmId string) func(*tls.ClientHelloInfo) (*tls
 
 			certsEndpoint := "certs"
 			csrEndpoint := "csrs"
-			endpoint := fmt.Sprintf("%s/%s/%s", certsEndpoint, csrEndpoint, cvmId)
+			endpoint := fmt.Sprintf("%s/%s/%s/%s", domainId, certsEndpoint, csrEndpoint, cvmId)
```

Comment:
let's import and use sdk directly. Import sdk in main.go if caURL is provided and pass the sdk to internal or where needed. if not provided use the fallback. we don't need to duplicate sdk methods in am-certs